src/commands/add.js

```diff
@@ -59,6 +59,12 @@ Examples:
     process.exit(exitCodes.OK)
   }
 
+  // we don't (yet) support scoped npm packages, so bail
+  if (parameters.second.startsWith('@')) {
+    print.error(`Error: Ignite CLI doesn't support prefixed npm packages yet, sorry about that. Try it without the prefix.`)
+    process.exit(1)
```

Comment:
Can you use a code from [exitCodes.js](https://github.com/infinitered/ignite/blob/master/src/lib/exitCodes.js)?